Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -160,10 +160,50 @@ IVolume::ECountry CVolumeWiiCrypted::GetCountry() const
 	if (!m_pReader)
 		return COUNTRY_UNKNOWN;
 
-	u8 country_code;
-	m_pReader->Read(3, 1, &country_code);
+	u8 country_byte;
+	if (!m_pReader->Read(3, 1, &country_byte))
+	{
+		return COUNTRY_UNKNOWN;
+	}
+
+	IVolume::ECountry country_value = CountrySwitch(country_byte);
+
+	u32 region_code;
+	if (!ReadSwapped(0x4E000, &region_code, false))
+	{
+		return country_value;
+	}
 
-	return CountrySwitch(country_code);
+	switch (region_code)
+	{
+		case 0:
```

Comment:
I don't really care one way or the other, but is this actually mentioned in the style guide?